.github/workflows/check-version.yaml

```diff
@@ -0,0 +1,102 @@
+name: Check Version Update
```

Comment:
Would be nice to say when this runs, what triggers it, what it does right at the top as a comment (same for the other .yamls)